.circleci/config.yml

```diff
@@ -175,23 +175,27 @@ jobs:
       - run:
           name: Set Python Version
           command: |
+            [ -d "/home/circleci/code/venv" ]
             pyenv versions
             pyenv global 3.6.5
             pyenv versions
       - run:
           name: Virtual Environment
           command: |
+            [ -d "/home/circleci/code/venv" ]
             python -m venv ~/code/venv
             echo "source ~/code/venv/bin/activate" >> $BASH_ENV
       - run: pip install --upgrade pip
       - run:
           name: Install Requirements
           command: |
+            [ -d "/home/circleci/code/venv" ]
```

Comment:
Wouldn't this always exit, because `Virtual Environment` would actually create this, but it would still be "empty"?

Reply:
neinei, and I realise this was stupid anywho